README.md

```diff
@@ -783,73 +787,68 @@ Here are a few ways you might hear of or see in documents:
 
 ### Classes of stock
 
-🄳 Often investors ask for rights to be paid back first in exchange for their investment.
+🄳 Investors often ask for rights to be paid back first in exchange for their investment.
 The way these different rights are handled is by creating different
-[**classes of stock**](https://www.upcounsel.com/classes-of-stock), each with its own.
-These are also sometimes called
-[**classes of shares**](https://www.investopedia.com/terms/c/class.asp) (though that term has
-another meaning in the context of mutual funds).
+[**classes of stock**](https://www.upcounsel.com/classes-of-stock). (These are also sometimes
+called [**classes of shares**](https://www.investopedia.com/terms/c/class.asp), though that term
+has another meaning in the context of mutual funds.)
 
 🄳 Two important classes of stock are
 [**common stock**](https://en.wikipedia.org/wiki/Common_stock) and
 [**preferred stock**](https://en.wikipedia.org/wiki/Preferred_stock). In general, preferred stock
-is stock that has rights, preferences, and privileges that common stock does not have.
+has
+[“rights, preferences, and privileges”](https://www.americanbar.org/publications/blt/2014/01/04_bigler.html)
+that common stock does not have.
 Typically, investors get preferred stock, and founders and employees get common stock (or
 an [option](#stock-options) to buy common stock).
 
-The exact number of classes of stock and the differences can vary company to company, and,
-in a startup, at [each round of funding](#stages-of-a-startup). If “rights, preferences, and
-privileges” sounds like confusing legalese,
-[it is](https://www.americanbar.org/publications/blt/2014/01/04_bigler.html). We’ll summarize
-the key points, though.
+The exact number of classes of stock and the differences between them can vary company to
+company, and, in a startup, these can vary at [each round of funding](#stages-of-a-startup).
 
-☝️ You’ll also hear the term
+☝️ Another term you’re likely to hear is
 [**founders’ stock**](http://www.alleywatch.com/2013/08/what-is-founders-stock-legally/), which is
 ([usually](https://lsvp.wordpress.com/2008/09/15/what-entrepreneurs-need-to-know-about-founders-stock/))
 common stock allocated at a company’s formation, but otherwise doesn’t have any different
 rights from other common stock.
 
-Now some details on how this works:
+Some key insights on all this:
```

Comment:
Although preferred stock rights are too complex to cover fully, we can give a few key details:

Reply:
(Most of these are terms or concepts, not insights)

Reply:
"Insights" works in the idea that there are many complexities and we are penetrating to the key concepts. That is its function.

Reply:
Fixed.